studio/components/interfaces/Database/Tables/ColumnList.tsx

```diff
@@ -25,9 +25,9 @@ const ColumnList: FC<Props> = ({
   const { meta } = useStore()
   const [filterString, setFilterString] = useState<string>('')
   const columns =
-    filterString.length === 0
+    (filterString.length === 0
       ? selectedTable.columns
-      : selectedTable.columns.filter((column: any) => column.name.includes(filterString))
+      : selectedTable.columns!.filter((column: any) => column.name.includes(filterString))) ?? []
```

Comment:
Don't know if it's the correct pattern to follow

Reply:
Using `!` will crash at runtime if the columns is actually undefined. I've updated the undefined check to use `?`.